Rating/rating.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rating</title>
+    <link rel="stylesheet" href="./rating.css">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+</head>
+<body>
+    <div class="rating">
+        <i class="fa fa-star filled"></i>
+        <i class="fa fa-star filled"></i>
```

Comment:
This is good enough and in react it will work perfectly but will recommend to make this more dynamic just for exploration, so that it can be changed according to the clicked